### DISCUSSION TITLE
关于mentor制度的讨论

### DISCUSSION THREAD

**Discussion by georgeliuyu** [icpdao/public#3] [General]
在mentor奖励机制的基础上，做如下几个规则：
1、任何人，必须由icpper邀请，才能成为icpper；
2、邀请人自动成为申请人的mentor；
3、每个有icpper有两个邀请名额；
4、被邀请的icpper一旦成为别人的mentor，则所占用的邀请名额自动释放；
5、每个icpper做贡献获得的token奖励的10%，分配给其上级mentor（50%、25%、13%、6%、3%、2%、1%）
大家可以就此进行讨论，特别是提出反对意见的角度。

**Comment by georgeliuyu** [comment]
缺点：
1、当所有的活跃icpper的名额都用完，而非活跃icpper有没有邀请人时，社区进入死亡模式；
2、对于公共宣传引流的新人不能一次通过登录成为icpper，流失率很高；
3、正好有一个很重要的朋友想成为icpper，而你的名额已经用完了。

**Reply by bqx619** [reply]
1. icpper stakes his community token to get invited, and the invitee's token is proportionally distributed to the person who staked the token.
1. icpper can redeem his or her token, after which the invitation relationship will be dissolved and the invitee will no longer be rewarded with token allocation, but can re-invite others by pledging again
1. invitation codes number can be determined by the number of tokens pledged by the icpper (upper-lower-limits)
1. person who wants to join can contact an existing person to invite him (stake token), and the person can evaluate whether he should invite him or not (whether he can bring enough rewards)
1. can a person who wants to join buy a token pledge to get invited directly?

---


1. icpper 质押自己的社区 token 获得邀请权, 被邀请人 token 按照比例分配给质押了 token 的人
1. icpper 可以赎回自己的 token, 赎回后邀请关系解散, 不再享有被邀请人的 token 分配奖励, 但可以通过再次质押的方式重新邀请别人
1. 邀请码数量可以根据 icpper 质押的 token 数量决定(上下限)
1. 希望加入的人员可以联系已有的人员邀请自己(质押 token), 人员可以变相评估自己是否应该邀请他(能否带来足够的奖励)
1. 希望加入的人员是否可以购买 token 质押直接让自己获得邀请?

**Reply by georgeliuyu** [reply]
很棒，大家有进步

**Reply by georgeliuyu** [reply]
可以在这个基础上进一步讨论

**Reply by georgeliuyu** [reply]
Can further explain the fifth point @bqx619 
have a disadvantage: icpper without token, can not invite others (unless he buys token)

可以进一步解释一下第五点 @bqx619 
有个缺点：没有token的icpper，不能邀请别人（除非购买token）

**Reply by georgeliuyu** [reply]
How the number of pledged tokens is determined？
质押token的数量如何确定？

**Reply by bqx619** [reply]
> Can further explain the fifth point @bqx619
> have a disadvantage: icpper without token, can not invite others (unless he buys token)

目前想到的第五点是基于
1. 如果我不想自己的奖励被分配, 那么我先买一些 token 把自己给"赎回", 这适用于长期贡献者
2. 对于只计划参与一两次的, 更适合找别人要邀请, 当然, 别人会评估你
3. 确实存在需要有 token 才能邀请别人, 但是可以变相刺激贡献也避免一味的希望通过邀请别人来获得 token

<details>
<summary>English</summary>

The fifth point that comes to mind is based on
1. if I don't want my rewards to be distributed, then I buy some tokens first to "redeem" myself, this applies to long-term contributors
2. for those who only plan to participate once or twice, it's better to ask for an invitation from someone else, who will, of course, evaluate you
3. It is true that you need a token to invite others, but it can be a disguised incentive to contribute and avoid the desire to invite others to get a token.
</details>

> How the number of pledged tokens is determined？

理论上, 质押了 token 后, token 会产生持续收益
理论上, 邀请新人会对 icpdao 产生新的运营/治理, 所以需要让 icpdao 获得奖励
或许, 可以质押 icp 而不是社区自己的 token, 由 icp 的持有者们, 投票确定需要的质押数量
或许, 可以质押社区自己的 token, 由社区 token 持有者投票, 然后社区 token 需要按原定质押给 icpdao

<details>
<summary>English</summary>

Theoretically, after pledging a token, the token will generate continuous revenue
Theoretically, inviting new people would create new operations/governance for the icpdao, so the icpdao would need to be rewarded
Perhaps, instead of the community's own token, the community could pledge the icp, and the holders of the icp would vote on the number of pledges needed
Perhaps, the community could pledge its own token, have the community token holders vote, and then the community token would need to be pledged to the icpdao as intended
</details>

**Reply by georgeliuyu** [reply]
是否与引进人才的急迫程度与质押的数量有关

**Reply by georgeliuyu** [reply]
可能的缺点：
1、巨鲸效应；（对比原方案）
2、没有隔级传播压力，

**Comment by fushang318** [comment]
是否可以这样，如果有名额裂变失败时，可以允许 mentor 和这个名额 交割，完全切断关系，然后再申请新的名额，根据社区需求来调整这种做法的频率和额度

**Reply by georgeliuyu** [reply]
有趣的想法

**Reply by georgeliuyu** [reply]
那么被切断的icpper，就没有mentor了

**Reply by fushang318** [reply]
对，就没有了

**Reply by georgeliuyu** [reply]
> 对，就没有了

解除mentor关系的icpper，10%的token还要扣除吗？

**Comment by georgeliuyu** [comment]
将mentor的关系，铸造成为NFT，进而可以进行交易的想法非常cool，逻辑是：1、mentor的关系是有价值的；2、能推动icp token持有和质押，进而推动token的价格上涨；3、在早期有强大的病毒推广效应，甚至会出现专业的推广组织和机构。
负面：一旦将mentor商品化，价格化，mentor关系将不再具有精神关系（特别是在今天这个失去信仰的时代）
这个问题是否可以放到以后社区去讨论和决策
The idea of casting the mentor relationship as NFT, which can then be traded is very COOL, the logic is: 1, the mentor relationship is valuable; 2, can promote icp token holding and pledging, which in turn will drive up the price of token; 3, in the early days there is a strong viral promotion effect, and even professional promotion organizations and institutions will emerge.
Negative: Once the mentor is commoditized and priced, the mentor relationship will no longer have a spiritual relationship (especially in today's era of losing faith)
Whether this issue can be put to the community to discuss and decide later

**Comment by georgeliuyu** [comment]
**mentor的核心作用：指导和帮助所属的contributor发现和参与贡献项目。**
The core role of the mentor is to guide and help the affiliated contributors to identify and participate in contributing projects.

**Comment by georgeliuyu** [comment]
1.  新用户登录，没有邀请：**在icpdao的世界中，每个人都会有一个mentor引导着你前行**  选择：申请/自己找
2. icpper，发起邀请：**我邀请@nancy进入icpdao的世界，在这里不管你是有疑问或者困难，我承诺给予支持，如果非我力所能及，也会向我的mentor需求帮助**   选择：是的，我承诺
3. 新用户，有邀请：**@bqx 会成为你在icpdao的mentor，并承诺你所需要的帮助，如ta力有所不及，也会寻找ta的mentor支持**  选择：谢谢，  **我承诺有问题或者遇到困难是，一定会请求@bqx的帮助**  选择：是的，我承诺

**Comment by georgeliuyu** [comment]
mentor v1：
在icpdao的世界中，每个参与者都一定会有一个mentor帮助和指导ta，其规则如下：
- 1、任何人，必须由icpper（包含pre-icpper）邀请，才能成为pre-icpper；
- 2、邀请人自动成为其mentor；
- 3、每个mentor最多只能同时指导两个pre-icpper；
- 4、pre-icpper一旦成为别人的mentor、或做过一次icp贡献，则自动转变为正式icpper；
- 5、mentor有权取消与pre-icpper的关系；
- 6、每个icpper做贡献获得的token奖励的5%作为mentor奖励，分配给其上级mentor（50%、25%、13%、6%、3%、2%、1%）
- 7、没有mentor的icpper的mentor奖励，直接进入icpdao的staking pool

**Comment by georgeliuyu** [comment]
- 承诺：
    - 新用户登录，没有邀请：
        - **在icpdao的世界中，每个人都会有一个mentor引导着你前行** 选择：申请/自己找
            - ？ icpper是icpdao世界中正式成员
    - 新用户，有邀请：
        - **欢迎来到icpdao的世界，@bqx愿意成为你的mentor**
        - **并承诺给予你所需要的帮助，如力所不及，还会向ta的mentor求助** 选择：好的，接受@bqx成为我的mentor， 
        - **恭喜你正式进入icpdao的世界，并成为pre-icpper，一旦你做了一次贡献、或者成为别人的mentor，就会自动转为正式icpper**
            - ？ icpper是icpdao世界中正式成员，pre-icpper是需要重点帮助的新成员
        - **在这之前，你是 @bqx 最重点关心和帮助的两个人之一** 
            - ？一个mentor同时最多只能支持2为pre-icpper
        - **也请你坦然接受这份帮助，你将来做的每次贡献，mentor也将受益** 
            - ？每个icpper做贡献获得的token奖励的5%作为mentor奖励，将自动分配给其一系列的上级mentor（50%、25%、13%、6%、3%、2%、1%）
        - **同时也希望你能成为mentor，帮助别人** 
    - icpper，发起邀请：
        - **你邀请@nancy进入icpdao的世界，成为其mentor**
        - **并承诺给予Ta所需要的帮助，如力所不及，也会向自己的mentor：@liu求助** 选择：是的，我承诺
    -

**Reply by georgeliuyu** [reply]
原文：https://roamresearch.com/#/app/icpdao/page/wyFaQSnpj
